testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication.py

```diff
@@ -609,9 +644,14 @@ def test_CBL_tombstone_doc(params_from_base_test_setup, num_of_docs):
     sg_client = MobileRestClient()
 
     # Modify sync-gateway config to use no-conflicts config
+    if x509_cert_auth:
+        persist_cluster_config_environment_prop(cluster_config, 'x509_certs', True)
+    else:
+        persist_cluster_config_environment_prop(cluster_config, 'x509_certs', False)
     c = cluster.Cluster(config=cluster_config)
     c.reset(sg_config_path=sg_config)
 
+    chan_cache_tombstone_revs = None
```

Comment:
why this declaration needed?

Reply:
because flake8 gave warning that variable may not be declared as it has value assigned in "if" condition below. So I have assigned None to assuage the warning